apps/cowswap-frontend/src/modules/tokensList/pure/TokenInfo/index.tsx

```
@@ -1,7 +1,12 @@
 import { TokenWithLogo } from '@cowprotocol/common-const'
+import { useCopyClipboard } from '@cowprotocol/common-hooks'
 import { TokenLogo } from '@cowprotocol/tokens'
 import { TokenName, TokenSymbol } from '@cowprotocol/ui'
 
+import { getEtherscanLink, shortenAddress } from 'libs/common-utils/src/legacyAddressUtils'
```

ESLint

[error] 'getEtherscanLink' is defined but never used.

(unused-imports/no-unused-imports)